dbm/wrapper.cpp

```diff
@@ -4,151 +4,253 @@
 #include "dbm/dbm.h"
 #include "hash/compute.h"
 
-extern "C"
+#define DBM(I, J) dbm->const_dbm()[(I)*dim + (J)]
+#define CDBM dbm->const_dbm()
```

Comment:
The macro was simply copied from the library internals, they were probably used because it was in the c-part of the code. dbm.c: 
```
/* More readable code with this */

#define DBM(I, J)  dbm[(I)*dim + (J)]
#define SRC(I, J)  src[(I)*dim + (J)]
#define DST(I, J)  dst[(I)*dim + (J)]
#define DBM1(I, J) dbm1[(I)*dim + (J)]
#define DBM2(I, J) dbm2[(I)*dim + (J)]
``` 
I was unaware of the ()-operator i agree and I have changed the code to use that instead